.github/workflows/e2e.yml

```
@@ -45,6 +46,16 @@ jobs:
           --health-interval=10s
           --health-timeout=5s
           --health-retries=5
+      valkey:
+        image: valkey/valkey:8.1.1
+        command: "valkey-server"
```

actionlint (1.7.7)

unexpected key "command" for "services" section. expected one of "credentials", "env", "image", "options", "ports", "volumes"

(syntax-check)